proxpn

```diff
@@ -136,21 +136,28 @@ TCP_EXIT_NODES[SanJose]=179.48.248.104
 
 UDP_EXIT_NODES[FREE]=196.52.21.65
 UDP_EXIT_NODES[NYC]=196.52.16.2
+UDP_EXIT_NODES[NYC2]=196.52.17.68
 UDP_EXIT_NODES[Miami]=196.52.22.2
+UDP_EXIT_NODES[Miami2]=196.52.23.66
 UDP_EXIT_NODES[Seattle]=196.52.18.2
+UDP_EXIT_NODES[Seattle2]=196.52.19.241
 UDP_EXIT_NODES[Chicago]=50.7.1.67
+UDP_EXIT_NODES[Chicago2]=196.52.21.225
 UDP_EXIT_NODES[Toronto]=162.253.128.67
 UDP_EXIT_NODES[HongKong]=43.251.159.25
```

Comment:
That's good you left the Hong Kong ones in there, in case they work for someone else.  Don't want to lose information :).